apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -212,6 +232,19 @@ export default function Default() {
                           height: "100%",
                         }}
                       >
+                        <FeatureToggle feature="protectServicesDesktop">
+                          <Switch>
+                            <Route
+                              exact
+                              path="/recover/:appId"
+                              render={(props: RouteComponentProps<RecoverComponentParams>) => (
```

Comment:
can you make sure to rebase with latest develop? We recently improved the typescript and we use `component` most of the time, this will need to pass typescript and we may have reinforce the rules (see on develop https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/renderer/Default.tsx#L248 )

Thanks

Reply:
I made the change 🙂